source/_integrations/tuya.markdown

```diff
@@ -36,38 +50,51 @@ The Tuya integration allows integrating all Powered by Tuya devices you have add
 - **Scene**: Smart scene.
 - **Switch**: Switch, socket, power strip, smart kettle, breaker, pet water feeder, air purifier, diffuser.
 
-## Quick installation and startup
-
-### Configure on the Tuya IoT Platform
+## Configuration of the Tuya IoT Platform
 
 #### Prerequisites
 
 You have created an account for [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
 
+You have created an account for the [Tuya IoT Platform](https://iot.tuya.com/).
```

Comment:
That is what the above line already says?

Reply:
The line above is for the app, this line is for the IoT platform, you need both (no the same account)